src/ui/map.js

```diff
@@ -1144,9 +1144,29 @@ class Map extends Camera {
      * {@link CanvasSourceOptions}.
      * @fires source.add
      * @returns {Map} `this`
-     * @see [Draw GeoJSON points](https://www.mapbox.com/mapbox-gl-js/example/geojson-markers/)
-     * @see [Style circles using data-driven styling](https://www.mapbox.com/mapbox-gl-js/example/data-driven-circle-colors/)
-     * @see [Set a point after Geocoder result](https://www.mapbox.com/mapbox-gl-js/example/point-from-geocoder-result/)
```

Comment:
ha, I just discovered this independently, was about to submit my PR when I noticed this.
Since https://github.com/mapbox/mapbox-gl-js/commit/6e6a0057bd4d6db3757c8775821115c44e778987#diff-44befe964c5c5c476298e33cc55ce3e7 theo geocoder example no longer uses addSource.